src/SFA.DAS.ApplyService.InternalApi/Controllers/RoatpGatewayOrganisationChecksController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Apply;
+using SFA.DAS.ApplyService.Domain.Ukrlp;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class RoatpGatewayOrganisationChecksController : Controller
+    {
+        private readonly IInternalQnaApiClient _qnaApiClient;
+        private readonly ILogger<RoatpGatewayOrganisationChecksController> _logger;
+
+        public RoatpGatewayOrganisationChecksController(IInternalQnaApiClient qnaApiClient, ILogger<RoatpGatewayOrganisationChecksController> logger)
+        {
+            _qnaApiClient = qnaApiClient;
+            _logger = logger;
+        }
+
+
+        [HttpGet("Gateway/Page/OrganisationAddress/{applicationId}")]
```

Comment:
I think this route needs to be /Roatp/Gateway/{applicationId}/Page/OrganisationAddress to work with current submaster properly

Reply:
It consistent with /Gateway/Page/CommonDetails/{applicationId}/{pageId}/{userName}

Reply:
/Gateway/{applicationId}/OrganisationAddress and /Gateway/{applicationId}/IcoNumber used